.github/workflows/concretize.yaml

```diff
@@ -35,14 +26,34 @@ jobs:
       - name: install-env
         uses: ./.github/actions/setup-spack-stack
         with:
-          specs: ${{ env.mac_os_specs }}
+          template: jedi-ufs-all
           compiler: ${{ matrix.compiler }}
           mpi: openmpi
           path: ${{ github.workspace }}
           concretize_only: 'true'
           site: macos.default
 
-  linux_concretize:
+  macos_gcc_concretize:
+    strategy:
+      matrix:
+        compiler: [gcc@9.4.0, gcc@10.3.0]
+    runs-on: macos-latest
+    steps:
+      - name: checkout
+        uses: actions/checkout@v2
+        with:
+          submodules: true
+
+      - name: install-env
+        uses: ./.github/actions/setup-spack-stack
+        with:
+          template: jedi-ufs-all
+          compiler: ${{ matrix.compiler }}
```

Comment:
I like that you are doing this for two compiler versions (same for clang and apple-clang above)!